client/my-sites/theme/main.jsx

```diff
@@ -41,7 +41,8 @@ import {
 	isThemePremium,
 	isThemePurchased,
 	getThemeRequestErrors,
-	getThemeForumUrl
+	getThemeForumUrl,
+	isWpcomTheme as isThemeWpcom,
```

Comment:
>  Can we simply call the prop isWpcom

I agree, the aliasing is bunk. But how would we know if `isWpcom` refers to the site or the theme? This is a common problem we have—if both the selector and prop are obviously named, we can't destructure the prop.